omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -102,7 +109,11 @@ public Object upload(MultipartFile file, RequestContext context) throws Response
 		Provider provider = Context.getProviderService().getProviderByUuid(context.getParameter("provider"));
 		String fileCaption = context.getParameter("fileCaption");
 		String instructions = context.getParameter("instructions");
+		String image = context.getParameter("cameraImage");
```

Comment:
Let's do:
```java
String base64Content = context.getParameter("base64Content");
```
The reason being that Base64 can be used to encode pretty much anything, not just images.